docs/pages/enroll-resources/database-access/enroll-managed-databases/oracle-exadata.mdx

```diff
@@ -0,0 +1,399 @@
+---
+title: Database Access with Oracle Exadata
+description: How to configure Teleport database access with Oracle Exadata.
+---
+
+(!docs/pages/includes/database-access/self-hosted-introduction.mdx dbType="Oracle Exadata"!)
```

Comment:
This will be quite confusing. The guide is added to "Enroll Cloud-Hosted Database Platforms" but 
```
self-hosted-introduction.mdx
```

Will reference to self-hosted 

 _The Teleport Database Service proxies traffic from database clients to **self-hosted** databases in your infrastructure. Teleport maintains a certificate authority for database clients. You configure your database to trust the Teleport database client CA, and the Teleport Database Service presents certificates signed by this CA when proxying user traffic. With this setup, there is no need to store long-lived credentials for **self-hosted** databases.

Meanwhile, the Teleport Database Service verifies **self-hosted** databases by checking their TLS certificates against either the Teleport database CA or a custom CA chosen by the user_

Reply:
Good catch, I'll make an update.